### ISSUE TITLE
Adding smtp server without user and password

### ISSUE THREAD

**Issue by troubleshootme** [towfiqi/serpbear#30]
I would like to use smtp-relay.gmail.com to send mail out but serpbear requires username and password for mail parameters but my relay auths via IP.

**Comment by towfiqi** [comment]
Thanks for reporting, will fix in next release.

**Comment by towfiqi** [comment]
@troubleshootme Upon looking into this, found out that even if you do use gmail, you would still need to use your username, password in the SMTP user and password field. You might still be blocked by google and you may have to go to https://www.google.com/settings/security/lesssecureapps and enable it.

**Comment by troubleshootme** [comment]
This is true for free Gmail, I am referring to Google's SMTP relay service for Google Workspace. They give you the option to use smtp-relay.gmail.com and whitelist server IP requiring no SMTP auth. I prefer to use that rather than enable less secure apps. Microsoft 365 also provides a similar service.

**Comment by towfiqi** [comment]
Fixed in latest release. Try pulling the latest docker image and you should be able to add your smtp details without user/pass.

**Comment by troubleshootme** [comment]
Thanks!


Tony Bruno / Consultant

***@***.*** / 205.876.3715

Bruno Computer Support

www.troubleshootme.com <https://troubleshootme.com/>

205.876.3715
